pkg/router/template/router.go

```diff
@@ -450,11 +458,38 @@ func (r *templateRouter) AddRoute(id string, route *routeapi.Route, host string)
 		}
 	}
 
+	var err error
```

Comment:
Hmm, ok - that would be require rework - will look into it when am back on Tuesday then.